### PR TITLE
{Release} Upgrade to Azure CLI 2.60.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

**Comment by yonzhan** [comment]
Release

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,111 @@
...
+
+* [BREAKING CHANGE] `az acr connected-registry create`: Mode default value change from ReadWrite to ReadOnly (#28807)
+* `az acr connected-registry create`: If data-endpoint disabled ask for confirmation to enable it instead of throwing an error (#28807)
+
+**AKS**
+
+* `az aks create/update`: Prompt warning during disablement about CR deletion (#28655)
+* `az aks create/update`: Udpate RP registration code to work on azure monitor subscription (#28607)
+* `az aks create/update`: Update to add default region for workspace creation in air gapped cloud (#28607)
+* Add option `--disable-windows-outbound-nat` for `az aks nodepool add` to add a Windows agent pool which the Windows OutboundNAT is disabled (#28806)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:19]:
```suggestion
* `az aks nodepool add`: Add parameter `--disable-windows-outbound-nat` to add a Windows agent pool which the Windows OutboundNAT is disabled (#28806)
```

Comment (yanzhudd) [inline src/azure-cli/HISTORY.rst:19]:
```suggestion
* `az aks nodepool add`: Add `--disable-windows-outbound-nat` to add a Windows agent pool which the Windows OutboundNAT is disabled (#28806)
```

File: src/azure-cli-core/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+2.60.0
+++++++
+* Fix #28737: Azure CLI tasks can now run longer than 5 minutes utilizing the token cache (#28747)
```
Comment (jiasli) [inline src/azure-cli-core/HISTORY.rst:8]:
```suggestion
* Fix #28737: Fix token cache for service principal authentication (#28747)
```

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,111 @@
...
+* `az batch pool create` Add `--max-unhealthy-instance-percent` argument to set the maximum percentage of the total VMs can be simultaneously unhealthy (#28784)
+* `az batch pool create` Add `--max-unhealthy-upgraded-instance-percent` argument to set the maximum percentage of upgraded VMs that can be found to be in an unhealthy state (#28784)
+* `az batch pool create` Add `--pause-time-between-batches` argument to set the wait time between batches in rolling OS upgrade (#28784)
+* `az batch pool create` Add `--prioritize-unhealthy-instances` argument to support upgrade all unhealthy VMs first (#28784)
+* `az batch pool create` Add `--rollback-failed-instances-on-policy-breach` argument to enable rollback failed instances to previous model if the Rolling Upgrade policy is violated (#28784)
+
+**CI**
+
+* Limit workflow target branches (#28706)
+
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:65]:
```suggestion
```

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,111 @@
...
+* `az netappfiles volume create`: Update maximum value for `--usage-threshold` to support large volumes (#28709)
+* `az netappfiles volume update`: Update maximum value for `--usage-threshold` to support large volumes (#28709)
+
+**Network**
+
+* `az network virtual-appliance inbound-security-rule`: Support for Permanent Inbound Security Rule (#28697)
+
+**RDBMS**
+
+* `az postgres flexible-server upgrade`: Add capability to perform major version upgrade to PG16. (#28687)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:95]:
```suggestion
* `az postgres flexible-server upgrade`: Add capability to perform major version upgrade to PG16 (#28687)
```

File: src/azure-cli/HISTORY.rst [hunk 5]
```diff
@@ -3,6 +3,111 @@
...
+* `az batch pool create` Add `--use-rolling-upgrade-policy` argument to support OS rolling upgrade policy (#28784)
+* `az batch pool create` Add `--enable-cross-zone-upgrade` argument to support cross zone OS upgrade (#28784)
+* `az batch pool create` Add `--max-batch-instance-percent` argument to set the maximum percent of total VMs that will be upgraded in one batch (#28784)
+* `az batch pool create` Add `--max-unhealthy-instance-percent` argument to set the maximum percentage of the total VMs can be simultaneously unhealthy (#28784)
+* `az batch pool create` Add `--max-unhealthy-upgraded-instance-percent` argument to set the maximum percentage of upgraded VMs that can be found to be in an unhealthy state (#28784)
+* `az batch pool create` Add `--pause-time-between-batches` argument to set the wait time between batches in rolling OS upgrade (#28784)
+* `az batch pool create` Add `--prioritize-unhealthy-instances` argument to support upgrade all unhealthy VMs first (#28784)
+* `az batch pool create` Add `--rollback-failed-instances-on-policy-breach` argument to enable rollback failed instances to previous model if the Rolling Upgrade policy is violated (#28784)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:60]:
```suggestion
* `az batch pool create`: Add `--upgrade-policy-mode` argument to support automatic OS upgrade (#28784)
* `az batch pool create`: Add `--enable-auto-os-upgrade` argument to enable automatic OS upgrade (#28784)
* `az batch pool create`: Add `--disable-auto-rollback` argument to disable OS image rollback feature (#28784)
* `az batch pool create`: Add `--defer-os-rolling-upgrade` argument to defer OS upgrades on the TVMs if they are running tasks (#28784)
* `az batch pool create`: Add `--use-rolling-upgrade-policy` argument to support OS rolling upgrade policy (#28784)
* `az batch pool create`: Add `--enable-cross-zone-upgrade` argument to support cross zone OS upgrade (#28784)
* `az batch pool create`: Add `--max-batch-instance-percent` argument to set the maximum percent of total VMs that will be upgraded in one batch (#28784)
* `az batch pool create`: Add `--max-unhealthy-instance-percent` argument to set the maximum percentage of the total VMs can be simultaneously unhealthy (#28784)
* `az batch pool create`: Add `--max-unhealthy-upgraded-instance-percent` argument to set the maximum percentage of upgraded VMs that can be found to be in an unhealthy state (#28784)
* `az batch pool create`: Add `--pause-time-between-batches` argument to set the wait time between batches in rolling OS upgrade (#28784)
* `az batch pool create`: Add `--prioritize-unhealthy-instances` argument to support upgrade all unhealthy VMs first (#28784)
* `az batch pool create`: Add `--rollback-failed-instances-on-policy-breach` argument to enable rollback failed instances to previous model if the Rolling Upgrade policy is violated (#28784)
```

File: src/azure-cli/HISTORY.rst [hunk 6]
```diff
@@ -3,6 +3,111 @@
...
+**Monitor**
+
+* `az monitor log-analytics workspace update`: Add parameter `--sku-name` (#28411)
+
+**NetAppFiles**
+
+* `az netappfiles`: Update to api-version 2023-07-01 (#28575)
+* `az netappfiles volume-group create`: Add `zones` argument to set Availability Zone for volume group volumes (#28709)
+* `az netappfiles volume create`: Update maximum value for `--usage-threshold` to support large volumes (#28709)
+* `az netappfiles volume update`: Update maximum value for `--usage-threshold` to support large volumes (#28709)
```
Comment (bebound) [inline src/azure-cli/HISTORY.rst:87]:
```suggestion
* `az netappfiles volume create/update`: Update maximum value for `--usage-threshold` to support large volumes (#28709)
```

File: src/azure-cli/HISTORY.rst [hunk 7]
```diff
@@ -3,6 +3,111 @@
...
+* Fix #28553: `az containerapp exec`: Fix the error of `inappropriate ioctl for device` (#28759)
+
+**Monitor**
+
+* `az monitor log-analytics workspace update`: Add parameter `--sku-name` (#28411)
+
+**NetAppFiles**
+
+* `az netappfiles`: Update to api-version 2023-07-01 (#28575)
+* `az netappfiles volume-group create`: Add `zones` argument to set Availability Zone for volume group volumes (#28709)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:85]:
```suggestion
* `az netappfiles volume-group create`: Add `--zones` argument to set Availability Zone for volume group volumes (#28709)
```

File: src/azure-cli/HISTORY.rst [hunk 8]
```diff
@@ -3,6 +3,111 @@
...
+
+* `az aks create/update`: Prompt warning during disablement about CR deletion (#28655)
+* `az aks create/update`: Udpate RP registration code to work on azure monitor subscription (#28607)
+* `az aks create/update`: Update to add default region for workspace creation in air gapped cloud (#28607)
+* Add option `--disable-windows-outbound-nat` for `az aks nodepool add` to add a Windows agent pool which the Windows OutboundNAT is disabled (#28806)
+
+**App Service**
+
+* `az webapp config container set`: Remove docker prefix and rename container related parameters (#28621)
+* `az webapp create` and `az webapp deployment slot create`: Rename `--docker-registry-server-user` and `--docker-registry-server-password` to `--container-registry-user` and `--container-registry-password` and add parameters `--container-image-name` and `container-registry-url` to deprecate `--deployment-container-image-name` (#28621)
```
Comment (yanzhudd) [inline src/azure-cli/HISTORY.rst:24]:
```suggestion
* `az webapp create` and `az webapp deployment slot create`: Rename `--docker-registry-server-user` and `--docker-registry-server-password` to `--container-registry-user` and `--container-registry-password` (#28621)
* `az webapp create` and `az webapp deployment slot create`: Add parameters `--container-image-name` and `container-registry-url` to deprecate `--deployment-container-image-name` (#28621)
```

File: src/azure-cli/HISTORY.rst [hunk 9]
```diff
@@ -3,6 +3,111 @@
...
+* `az containerapp env create`: Support `--dapr-connection-string` to set application insights connection string used by Dapr to export service to service communication telemetry (#28625)
+* Fix #28553: `az containerapp exec`: Fix the error of `inappropriate ioctl for device` (#28759)
+
+**Monitor**
+
+* `az monitor log-analytics workspace update`: Add parameter `--sku-name` (#28411)
+
+**NetAppFiles**
+
+* `az netappfiles`: Update to api-version 2023-07-01 (#28575)
```
Comment (kairu-ms) [inline src/azure-cli/HISTORY.rst:84]:
```suggestion
```

**Review by zhoxing-ms — APPROVED** [review]

**Review by evelyn-ys — APPROVED** [review]

**Review by necusjz — APPROVED** [review]

**Review by jiasli — APPROVED** [review]

**Review by kairu-ms — APPROVED** [review]

**Review by yanzhudd — APPROVED** [review]

**Review by wangzelin007 — APPROVED** [review]

**Review by bebound — APPROVED** [review]

**Review by Moazzem-Hossain-pixel — COMMENTED** [review]
Need support

**Pull request merged**